private-templates-service/client/src/components/Common/Tags/Tags.tsx

```diff
@@ -1,10 +1,10 @@
-import React from 'react';
-import { connect } from 'react-redux';
+import React from "react";
```

Comment:
For this file, is it possible to uncommit your changes and re-apply them with your linter disabled? Khang modified this file heavily and the conflicts will be difficult to solve.

Reply:
Yup, done